services/latex_service.rb

```diff
@@ -17,8 +17,8 @@ def self.render?(message, path, file)
     # -T is image size
     # -o is output file
     if did_comp
-      system("dvipng -q* -D 300 -T tight #{File.join(path, file)}.dvi -o #{File.join(path, file)}.png")
-    else 
+      system("convert -density 300 -flatten #{File.join(path, file)}.dvi #{File.join(path, file)}.png >>/dev/null")
+    else
       return false
     end
```

Comment:
Not sure why GitHub isn't letting me make a commit suggestion, but normally you can just reduce a statement like this to this:
```
return false unless did_comp
system("convert -density 300 -flatten #{File.join(path, file)}.dvi #{File.join(path, file)}.png >>/dev/null")
```